src/processor/Package.ts

```diff
@@ -60,3 +75,17 @@ export class Package {
     }
   }
 }
+
+function isDuplicateDefinition(
+  array: NamedExportable[],
+  resource: NamedExportable,
+  type: string
+): boolean {
+  if (array.find(e => e.name === resource.name)) {
+    logger.error(
+      `Encountered ${type} with a duplicate name, ${resource.name}, which GoFSH cannot make unique. Fix the source file to resolve this error. The definition will still be written to a .fsh file.`
```

Comment:
> The definition will still be written to a .fsh file.

What does this mean?  I think you mean that only one of the definitions will be written, but I only know that based on your implementation.  Can you clarify this?
> Only one of the duplicate definitions will be written to a .fsh file. Remaining duplicates will be skipped.

Reply:
Oops yes, the error message is wrong and nothing is written to a FSH file. It should say 

> The definition will not be written to a .fsh file.

Reply:
I think the error message should be more clear in 1dc1f5e. It should add the definition and write to a FSH file now though.